README.md

```diff
@@ -1,7 +1,19 @@
-# Deprecation of APIs or API endpoints
+# Reform API standards
 
```

Comment:
The readme needs some love imho, for example adding a paragraph explaining that this is a library containing a set of components that can be used in a **Spring Boot** app, but I'd rather leave that to a separate pull request, not dedicated to adding this new filter.